command/gems/gemfile_lock.go

```diff
@@ -0,0 +1,89 @@
+package gems
+
+import (
+	"fmt"
+	"regexp"
+	"strings"
+)
+
+// Version contains bundler or fastlane version
+type Version struct {
+	Version string
+	Found   bool
+}
+
+// ParseVersionFromBundle  returns the specified grm version parsed from a Gemfile.lock on a best effort basis, for logging purposes only.
+//
+// for "fastlane" and the following Gemfile.lock example, it returns: ">= 2.0)"
+//   specs:
+//     CFPropertyList (3.0.0)
+//     addressable (2.6.0)
+//       public_suffix (>= 2.0.2, < 4.0)
+//     atomos (0.1.3)
+//     babosa (1.0.2)
+//     badge (0.8.5)
+//       curb (~> 0.9)
+//       fastimage (>= 1.6)
+//       fastlane (>= 2.0)
+//       mini_magick (>= 4.5)
+//     claide (1.0.2)
+func ParseVersionFromBundle(gemName string, gemfileLockContent string) (gemVersion Version, err error) {
+	relevantLines := []string{}
+	lines := strings.Split(gemfileLockContent, "\n")
+
+	specsStart := false
+	for _, line := range lines {
+		trimmed := strings.Trim(line, " ")
+		if trimmed == "" {
+			specsStart = false
+		}
+
+		if strings.Contains(line, "specs:") {
+			specsStart = true
+			continue
+		}
+
+		if specsStart {
+			relevantLines = append(relevantLines, line)
+		}
+	}
+
+	//     fastlane (1.109.0)
+	exp := regexp.MustCompile(fmt.Sprintf(`^%s \((.+)\)`, regexp.QuoteMeta(gemName)))
```

Comment:
NOTE: there should be a way to explicitly define that the first `)` should form the match to avoid having issues with lines like: `fastlane (1.109.0) # could be bumped to fastlane (1.110.0)`

Reply:
It should in this case, because ^ is matching the beginning of text or line.

Reply:
very nice